slam-libraries/viam-cartographer/src/slam_service/slam_service.h

```diff
@@ -258,6 +258,11 @@ class SLAMServiceImpl final : public SLAMService::Service {
     // the respective member variables.
     void BackupLatestMap();
 
+    // If using the LOCALIZING action mode, cache a copy of the map before
+    // beginning to process data. If cartographer fails to do this,
+    // terminate the program
+    bool CacheMapInLocalizationMode();
```

Comment:
[minor]

```suggestion
    // terminate the program.
    void CacheMapInLocalizationMode();
```

Reply:
was already making changes so I added this to my commit